crates/wit-bindgen/src/lib.rs

```diff
@@ -63,6 +68,18 @@ impl Opts {
     }
 }
 
+#[cfg(feature = "clap")]
```

Comment:
Oh all this clap stuff is a holdover from copying over from the wit-bindgen repository, I think all of it can be removed since `clap` isn't even a dependency in this repo.

Reply:
Oh! I didn't even think to look. Deleted.